src/telemetry/constants.ts

```diff
@@ -15,7 +15,11 @@ export enum EventName {
 
     OPEN_DATAVIEWER_FROM_VARIABLE_WINDOW_REQUEST = 'OPEN_DATAVIEWER_FROM_VARIABLE_WINDOW_REQUEST_EX',
     OPEN_DATAVIEWER_FROM_VARIABLE_WINDOW_ERROR = 'OPEN_DATAVIEWER_FROM_VARIABLE_WINDOW_ERROR_EX',
-    OPEN_DATAVIEWER_FROM_VARIABLE_WINDOW_SUCCESS = 'OPEN_DATAVIEWER_FROM_VARIABLE_WINDOW_SUCCESS_EX'
+    OPEN_DATAVIEWER_FROM_VARIABLE_WINDOW_SUCCESS = 'OPEN_DATAVIEWER_FROM_VARIABLE_WINDOW_SUCCESS_EX',
+
+    OPEN_DATAWRANGLER_FROM_OUTPUT_REQUEST = 'OPEN_DATAWRANGLER_FROM_OUTPUT_REQUEST_EX',
```

Comment:
I don't see an entry for this new telemetry in the index.ts? THat's where you'd put comments about the telemetry and what each event is for. It makes the generated telemetry.md file have more information.

Reply:
good catch, didn't notice that file and tried modifying the MD file directly